lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -510,8 +519,9 @@ def make_datatable(self):
 
         for course in self.get_courses():
             gdir = course.id.course
+            version = int(course.edited_on.strftime('%s'))
```

Comment:
I don't really understand your question here. Yes, I'm taking the unix timestamp of the last commit for use in the cache key. When the course is updated, the timestamp should change, meaning we get a different cache key. The old one will expire out of the cache via LRU and our newer entry will stay. 

There is no case that I'm aware of that will cause a version to repeat but mean two different entries.